approval_polls/urls.py

```diff
@@ -16,4 +16,7 @@
     url(r'^my-info/$', views.myInfo, name='my_info'),
     url(r'^set-user-timezone/$', views.set_user_timezone, name='set-user-timezone'),
     url(r'^invitation/(?P<pk>\d+)/$', views.DetailView.as_view(), name='invitation'),
+    url(r'^tag/(?P<tag>\w+)/$', views.taggedPolls, name='tagged_polls'),
```

Comment:
Currently this seems to be preventing space-separated tags eg. `New York` (or non-word special character tags) from being queried. 
It results in the error: 

> Reverse for 'tagged_polls' with arguments '(u'multiple words',)' and keyword arguments '{}' not found. 1 pattern(s) tried: [u'approval_polls/tag/(?P<tag>\\w+)/$']

I think we should be allowing single space separated tags. It would be great if you could add a test for it as well.